src/Database/SchemaCache.php

```diff
@@ -30,16 +30,16 @@ class SchemaCache
     /**
      * Schema
      *
-     * @var \Cake\Database\Schema\CachedCollection
+     * @var \Cake\Database\Schema\CachedCollection|\Cake\Database\Schema\Collection
```

Comment:
This change is making pslam unhappy https://travis-ci.org/cakephp/cakephp/jobs/458819610#L835